isis/src/cassini/objs/VimsCamera/VimsCamera.truth

```diff
@@ -141,8 +141,8 @@ DeltaSample = 0.000000000
 DeltaLine = 0.000000000
 
 For center pixel position ...
-Latitude off by: 0.0229595838205015
-Longitude off by: 0.0098872441018614
+Latitude off by: 0.0229595838120105
+Longitude off by: 0.0098872441060109
```

Comment:
For these unit tests you should updated the expected latitude and longitude values, not how much they are off by.